src/style.css

```diff
@@ -0,0 +1,75 @@
+html,
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  min-width: 1024px;
+  margin: 0 auto;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.header-menu {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.header-menu > li {
+  list-style-type: none;
+}
```

Comment:
remove this and write this rule inside `.header-menu--item`

Reply:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element